### PR TITLE
Adding Queries and Setting Up Unit Testing 

### LINKED ISSUE #58: As a department manager I want to produce a report summarising the number of people who speak the following languages (Chinese,English,Hindi,Spanish,Arabic) from the greatest number to smallest, including the percentage of the world population

### PR DESCRIPTION
**Pull request by MxmIv**
Resolves #58  and #56 

Some testing will need to be added

### CODE REVIEW

File: src/main/java/com/napier/sem/City.java [hunk 1]
```diff
@@ -1,12 +1,80 @@
...
+
+    // Updates the City Population
+    public void setPopulation(int population) {
+        this.population = population;
+    }
+    // gets the City Population
+    public int getPopulation() {
+        return population;
+    }
+}
```
Comment (tomgus1) [inline src/main/java/com/napier/sem/City.java:80]: Can a new line be added at the end off the file?

File: src/main/java/com/napier/sem/Country.java [hunk 2]
```diff
@@ -1,20 +1,79 @@
...
+    public int getPopulation() { return population; }
+
+    // Updates the Country Capital
+
+    public void setCapital(String capital) { this.capital = capital; }
+
+    // gets the Country Capital
+
+    public String getCapital() { return capital; }
+}
```
Comment (tomgus1) [inline src/main/java/com/napier/sem/Country.java:79]: As above

File: src/main/java/com/napier/sem/Population.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    public long getCountryPopulation() { return populationCountry; }
+
+    // Updates the Population of people living in Cities
+
+    public void setCityPopulation(long populationCity) { this.populationCity = populationCity; }
+
+    //gets the Population of people living in Cities
+
+    public long getCityPopulation() { return populationCity; }
+}
```
Comment (tomgus1) [inline src/main/java/com/napier/sem/Population.java:41]: As above

File: src/main/java/com/napier/sem/Country.java [hunk 4]
```diff
@@ -1,20 +1,79 @@
...
-    public double lifeExpectancy; // life expectancy of the danger
-    public double gnp; // GNP
-    public double gnpOld; //GNPOld
-    public String localName; //local name
-    public String governmentForm; // government form
-    public String headOfState; //head of state
-    public int capital; //capital
-    public String code2; //code 2
-}
+
```
Comment (tomgus1) [inline src/main/java/com/napier/sem/Country.java:3]: I think inline comments like before would be a lot more readable

File: src/main/java/com/napier/sem/Language.java [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    public long getPopulation() { return population; }
+
+    // Updates the Percentage of Worldwide speakers
+
+    public void setPercentage(float percentage) { this.globalPercentage = percentage; }
+
+    // Retrieves the Percentage of Worldwide speakers
+
+    public float getPercentage() { return globalPercentage; }
+}
```
Comment (tomgus1) [inline src/main/java/com/napier/sem/Language.java:40]: As above

File: src/main/java/com/napier/sem/Country.java (hunk 4 above)
Comment (MxmIv) [reply src/main/java/com/napier/sem/Country.java:3]: Andreas told me to have comments above methods, as it makes the code more readable

+ the code with inline comment is completely gone now

**Review by tomgus1 — APPROVED** [review]

**Review by tomgus1 — APPROVED** [review]

**Pull request merged**